views/clients/show.erb

```diff
@@ -8,92 +8,195 @@
     Description: <%= @client.description %> <br>
     <%= @client.active ? 'Active' : 'Inactive' %>
 </p>
+
 <% unless @projects.nil? %>
```

Comment:
should print something if projects are nil instead of letting the user see blank